lib/vanagon/platform/windows.rb

```diff
@@ -281,11 +282,11 @@ def add_repository(definition)
         commands = []
 
         if definition.scheme =~ /^(http|ftp|file)/
-          if File.extname(definition.path) == '.ps1'
-            commands << %(powershell.exe -NoProfile -ExecutionPolicy Bypass -Command 'iex ((new-object net.webclient).DownloadString(\"#{definition}\"))')
-          else
-            commands << %(C:/ProgramData/chocolatey/bin/choco.exe source add -n #{definition.host}-#{definition.path.gsub('/', '-')} -s "#{definition}" --debug || echo "Oops, it seems that you don't have chocolatey installed on this system. Please ensure it's there by adding something like 'plat.add_repository 'https://chocolatey.org/install.ps1'' to your platform definition.")
-          end
+          commands << if File.extname(definition.path) == '.ps1'
```

Comment:
We have a lot of these. We can rewrite them by splitting things into discrete methods or using ternary operators, or disable that particular Cop. I'm leaning towards "disable that particular Cop", but maybe doing so inline so we've got a searchable record of where the bodies are buried.